### PR TITLE
Single cell fix

### PR DESCRIPTION
**Pull request by denklewer**
Related to #8 
This is more clean version of the single cell script after reproducing.
Since filtering was not repeatable for original scripts, I've got another clusters. 
It has some problems of clarity for one of the clusters.

image

image

image

### CODE REVIEW

**Comment by denklewer** [comment]
I also used my "universal" optimization since select_x steps used are filtering dependent. But we can try to apply same learning rate manipulations for select_x

**Comment by denklewer** [comment]
But KNN is showing it more similar to the paper
image

**Review by almdudleer — APPROVED** [review]
Well I guess this works, some comments added, etc

**Comment by almdudleer** [comment]
Ok, let me try to fix the filtering to be as it is in the paper

File: figures/6_single_cell_hnsc.Rmd [hunk 1]
```diff
@@ -6,18 +6,30 @@ output:
     self_contained: yes
 ---
 
-```{r}
-source("../R/setup.R")
+# Setup
+```{r setup}
+# source("../R/setup.R")
```
[the remark this answers is not in the document]
Comment (denklewer) [reply figures/6_single_cell_hnsc.Rmd:11]: You are right, should be there, just accidental

**Comment by denklewer** [comment]
> Ok, let me try to fix the filtering to be as it is in the paper

@almdudleer how it's going?

**Pull request merged**